test/JDBC/expected/table_variable_xact_errors_isolation_snapshot.out

```diff
@@ -583,10 +583,6 @@ AS
     DELETE trgt FROM Discount trgt           -- Discount does not exist
     COMMIT
 go
-~~ERROR (Code: 2714)~~
```

Comment:
why this change?

Reply:
I am adding a drop of the function so the error does not exist anymore.

```
Message: Function 'usp_populatediscount' already exists with the same name
```